uint8 frequency
---
proc_mapping/PingerLocation pingerLocation